api/src/schemas.py

```diff
@@ -52,16 +57,17 @@ def exists(cls, collection):
 
 
 class DashboardCollection(Collection):
-    is_periodic: bool = Field(alias="dashboard:is_periodic")
+    is_periodic: bool = Field(default=False, alias="dashboard:is_periodic")
     time_density: Literal["day", "month", "year", "null"] = Field(
-        alias="dashboard:time_density", default="null"
+        default="null", alias="dashboard:time_density"
     )
     item_assets: Dict
+    extent: DatetimeExtent
 
     @validator("item_assets")
-    def cog_default_exists(cls, item_assets):
-        validators.cog_default_exists(item_assets=item_assets)
-        return item_assets
+    def cog_default_exists(cls, v):
+        validators.cog_default_exists(item_assets=v)
+        return v
```

Comment:
was there a special reason for this change? I'd argue that we should strive for descriptive naming instead of single-letter names.

Reply:
I've been trying to stick to the standard/examples in Pydantic's docs, but I'm not attached to that standard (and I only made that change to validators I touched, so it's not consistent anyways). How about we go with the standard: validator parameters should match their model field name, unless the validator applies to multiple fields (such as for root validators or the `set_ts_now` validator on line 110).